html.go

```diff
@@ -124,7 +124,7 @@ func (w *HTMLWriter) Newline() {
 
 // NewHTMLRenderer creates and configures an HTMLRenderer object, which
 // satisfies the Renderer interface.
-func NewHTMLRenderer(params HTMLRendererParameters) Renderer {
+func NewHTMLRenderer(params HTMLRendererParameters) *HTMLRenderer {
```

Comment:
I'm in favor of dropping it from this repo. That way we can focus more on Markdown parsing here.